# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libdc1394/example-clock-def.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libdc1394-1.0.0/examples/grab_partial_image.c.orig	2006-01-22 16:58:25.000000000 +0200
+++ libdc1394-1.0.0/examples/grab_partial_image.c	2006-01-22 16:59:27.000000000 +0200
@@ -213,7 +213,7 @@
     /*---------------------------------------------------------------------
      *  output elapsed time
      *---------------------------------------------------------------------*/
-    elapsed_time = (float)(times(&tms_buf) - start_time) / CLK_TCK;
+    elapsed_time = (float)(times(&tms_buf) - start_time) / CLOCKS_PER_SEC;
     printf( "got frame %d. elapsed time: %g sec ==> %g frames/second\n",
             i, elapsed_time, (float)i / elapsed_time);
   }
